python/feast_spark/pyspark/launchers/k8s/k8s.py

```diff
@@ -57,6 +62,10 @@ def _generate_job_id() -> str:
     )
 
 
+def _generate_scheduled_job_id(project: str, feature_table_name: str) -> str:
+    return f"feast-{project}-{feature_table_name}".replace("_", "-")
```

Comment:
is there any restrictions on name length? should we truncate here as well?

Reply:
Its 63 char for labels and 253 char for resource name. I removed the job id from label (since it's not being used anyway), and also truncate the string to be below 253 chars.